{"uuid": "4389265e-b032-4a49-a766-e4f63583e449", "children": ["16f1a51f-cbd5-4b2f-a687-484ae179f3c0"], "befores": [{"name": "Инициализируем списки ингредиентов", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'HelpersOnCheck' has no attribute 'get_ingredients'. Did you mean: 'check_ingredients'?\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lapka/Downloads/Diplom_3/conftest.py\", line 25, in setup_ingredients\n    ingredients = HelpersOnCheck.get_ingredients()\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1720077989986, "stop": 1720077989986}], "start": 1720077989986, "stop": 1720077990004}